arklib/Cargo.toml

```diff
@@ -8,7 +8,7 @@ name = "arklib"
 crate-type = ["cdylib", "staticlib"]
 
 [dependencies]
-arklib = { git = "https://github.com/ARK-builders/arklib", rev = "1b3a4519d17a35960bfde2d9ed25cb96f9f0" }
+arklib = { git = "https://github.com/ARK-builders/arklib", rev = "10c94d0b19b17e2d7b1662f8757242c6a26b5678" }
```

Comment:
@gwendalF : We need to merge the changes of this commit into `main` and replace this commit with `main` commit instead.

Reply:
@gwendalF this branch `MacNoId`? Create the PR please.

Reply:
do we need branch `MacCompilation` in `arklib`?